codecarbon/core/api_client.py

```diff
@@ -52,6 +52,14 @@ def __init__(
         self.conf = conf
         if self.experiment_id is not None:
             self._create_run(self.experiment_id)
+        import warnings
+
+        # FIXME: remove this warning in the future once the release is created
+        warnings.warn(
+            "If you use Codecarbon API, team id is being deprecated in favor of organization id in the next releases. Please update your code if needed.",
```

Comment:
Il faudrait un message plus générique, car on risque de casser plus de choses que la notion de Teams.
Par exemple : 
"Beta API will be reworked, and some features will be removed. If you have data persisted through the API, please be warned that it will be erased with the next API release".

Reply:
Done! [409485f](https://github.com/mlco2/codecarbon/pull/581/commits/409485fe46c3c87236797e76f067e88818691f09)